### DISCUSSION TITLE
File activation

### DISCUSSION THREAD

**Discussion by danielchalmers** [CommunityToolkit/Maui#1641] [Q&A]
Hi! I'm trying to make my app the default handler for a file extension. Is there an API or a proposal on this repo to help with that?

I was using the FilePicker but am trying to avoid the READ_EXTERNAL_STORAGE permission if possible.

**Comment by VladislavAntonyuk** [comment]
FilePicker is different feature
You need to change your manifest and add extensions your app can open.

**Reply by danielchalmers** [reply]
Thanks, is there any documentation to follow on this process? What do I change in the manifest? How do I add code to handle the activation?

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
